src/tests/skiff/calendar.test.ts

```diff
@@ -246,13 +258,6 @@ describe("Skiff Calendar", () => {
     // Find and click on the event card
     await getEventCard().then(async (card) => card?.click());
 
-    await sleep(2_000);
-
-    expect((await getConferenceValue())?.startsWith(TALK_BASE_URL)).toBe(true);
-
-    // Evaluate the button's visibility when a meeting is scheduled
-    expect(await getBraveTalkButton()).toBe(null);
```

Comment:
Both of these `expect` statements are sufficiently covered in the preceding test.